lib/image-diff.js

```diff
@@ -73,7 +73,7 @@ ImageDiff.createDiff = function (options, cb) {
     // DEV: According to http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=17284
     // DEV: These values are the total square root mean square (RMSE) pixel difference across all pixels and its percentage
     // TODO: This is not very multi-lengual =(
-    var resultInfo = stderr.match(/all: (\d+\.?\d*) \((\d+\.?\d*)\)/);
+    var resultInfo = stderr.match(/all: (\d+\.?\d*) \((.*?)\)/);
```

Comment:
Ah, interesting. We should probably isolate the parser to a separate function so we can test these individually. I imagine it will become very tedious to create a set of separate images for each of these cases.

As for the regexp, being explicit is always more useful than being implicit. Additionally, we should probably include examples of the different values in comments. The regexp should look like:

```js
/all: (\d+\.?\d*) \((\d+\.?\d*e?\-?\d*)\)/)
```

This allows for `0`, `0.1`, `0.1e-3`, and even unexpected cases like `1e3`, `1.0e3`.